cluster/crypto/Makefile

```diff
@@ -0,0 +1,56 @@
+
+.PHONY: all clean
+all: all.tar
+
+clean:
+	rm -f *.tar
+	rm -f *.pem
+	rm -f *.csr
+	rm -f ca.srl
+	rm -f openssl.cnf
+
+# config
+openssl.cnf: openssl.cnf.in san-extras
+	rm -f openssl.cnf
+	cat openssl.cnf.in >> openssl.cnf
+	cat san-extras >> openssl.cnf
+
+# root CA
+ca-key.pem:
```

Comment:
This was a hack-fest I started writing but never used: https://gist.github.com/aaronlevy/265897e547e80ba95202b6fad65bb07c

Not sure what issue you were running into with the wildcard targets/dependencies?